karapace/schema_registry_apis.py

```diff
@@ -903,22 +1061,29 @@ def write_new_schema_local(
         )
         self.r({"id": schema_id}, content_type)
 
-    async def _forward_request_remote(self, *, body, url, content_type, method="POST"):
+    async def _forward_request_remote(
```

Comment:
To keep the API tighter, this could have instead of full `request` just `authorization: Optional[str]`.

Reply:
I think passing around `request` simpler instead of querying `Authorization` header in every call invocation.